examples/attaching/attaching.go

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: BSD-2-Clause-FreeBSD
+//
+// Copyright (c) 2020 Florian Limberger <flo@purplekraken.com>
```

Comment:
Don't you want to assign the copyright for the example files to yourself? I didn't work on them ;)

Reply:
But you did worked on everything else :D 

I don't mind leaving it to the original author, really :)

Do you mind if we use this library in vk-jails to make jails something in Kubernetes? :)